config/routes.rb

```diff
@@ -0,0 +1,10 @@
+Rails.application.routes.draw do
+  namespace :api do
+    resources :greetings, only: [:index]
+  end
```

Comment:
- [ ] Based on the requirements of this project, the two apps should work together. Right now, the API endpoint for returning random greetings returns **null** hence could not connect properly with the client side. Kindly fix the issue


![Screenshot from 2022-12-01 19-25-38](https://user-images.githubusercontent.com/63087491/205131301-5832b2ea-a994-4167-a69f-6e618a71b19a.png)